plugins/in_winevtlog/pack.c

```diff
@@ -301,45 +301,45 @@ static void pack_string_inserts(struct winevtlog_config *ctx, PEVT_VARIANT value
             }
             break;
         case EvtVarTypeSByte:
-            flb_log_event_encoder_append_body_int8(ctx, values[i].SByteVal);
+            flb_log_event_encoder_append_body_int8(ctx->log_encoder, values[i].SByteVal);
```

Comment:
Are there any tests that need to be updated?

Reply:
I was going to, but there appear not to be any unit tests for this plugin. They would be a great thing to have though...

Reply:
Sorry for the inconvenience.... I'd tried to add that, however I wasn't able to handle it. Maybe, in the future, I can take a look on this.
Your work is really helpful for us. :+1: